woocommerce/woocommerce-admin
# Add currency defaults based on country

Fixes #3101.

This PR sets smart currency defaults based on country for a handful of our top countries and the EU, instead of defaulting to WooCommerce's default of `GBP`. If we don't support the country, USD is defaulted.

### Detailed test instructions:

* Go to the store details step and select a supported country, or an EU country.
* Continue to the next step.
* Observe the `batch` settings request, or view the currency settings under `WooCommerce > Settings` to make sure the defaults set.
* Select another country, and make sure USD defaults are used.
